models/questions.py

```diff
@@ -15,10 +15,10 @@
 
 # In SQL we can manually add the constraint
 # alter table questions add constraint name_bc_unique UNIQUE(name, base_course);
-try:
-    db.executesql('''alter table questions add constraint name_bc_unique UNIQUE(name, base_course)''')
-except:
-    pass
+# try:
+#     db.executesql('''alter table questions add constraint name_bc_unique UNIQUE(name, base_course)''')
+# except:
+#     pass
```

Comment:
If this is commented out, then new installations will not get this constraint which we want to preseve?  Why comment it out?